test/src/apidemos/expect-global/testExpect.js

```diff
@@ -48,7 +48,7 @@ describe('expect(element.<command>) - passed', function() {
     const testsPath = path.join(__dirname, '../../../apidemos/expect-global/expect.js');
     
     Mocks.elementNotSelected();
-    Mocks.elementProperty('0', 'className', {value: ['div-container']});
+    Mocks.elementProperty('0', 'className', {value: ['container']});
```

Comment:
This change is fixing a test which not reported earlier. The Nightwatch-test running inside this test has two expect out of which one should fail but without this change there would be two failures.
Nightwatch- Test: https://github.com/nightwatchjs/nightwatch/tree/main/test/apidemos/expect-global/expect.js